src/components/toolbox/infoBanner/infoBanner.js

```diff
@@ -9,12 +9,14 @@ const InfoBanner = ({
   name,
   infoLabel,
   infoMessage,
+  infoDescription,
   infoLink,
   className,
+  show,
   t,
 }) => {
   const [visibility, setVisibility] = useState(
-    !localStorage.getItem(name) ? 'visible' : 'hidden',
+    !localStorage.getItem(name) && show ? 'visible' : 'hidden',
   );
   const isLoggedIn = useSelector(state => (state.account && state.account.passphrase));
 
```

Comment:
@isalga I think the value of `visibility` can be of type boolean since it only has two states `visible` and `hidden`